modules/dag-l1/src/main/scala/org/tessellation/dag/l1/StateChannel.scala

```diff
@@ -125,6 +126,16 @@ class StateChannel[F[_]: Async: KryoSerializer: SecurityProvider: Random](
       storages.block.store(fb.hashedBlock).handleErrorWith(e => logger.debug(e)("Block storing failed."))
     }
 
+  private val sendBlockToL0: Pipe[F, FinalBlock, Unit] =
+    _.evalMap { fb =>
+      (for {
+        tips <- storages.block
+          .pullTips(appConfig.tips.maximumTipsCount)
+        // l0Peer <- storages.l0Cluster.getPeers.map(Random.shuffle).map(_.head)
+        // req <- p2PClient.l0DAGCluster.sendL1Output(L1Output(fb.hashedBlock.signed, tips))(l0Peer)
```

Comment:
are you going to continue on that in this PR?

Reply:
Yes